src/inheritAttributes.js

```diff
@@ -287,6 +287,45 @@ export const parseCaptionServiceMetadata = (service) => {
   }
 };
 
+/**
+ * A map callback that will parse all event stream data for a collection of periods
+ * DASH ISO_IEC_23009 5.10.2.2
+ * https://dashif-documents.azurewebsites.net/Events/master/event.html#mpd-event-timing
+ *
+ * @param {PeriodInformation} period object containing necessary period information
+ * @return a collection of parsed eventstream event objects with
+ */
+export const toEventStream = (period) => {
+  // get and flatten all EventStreams tags and parse attributes and children
+  return flatten(findChildren(period.node, 'EventStream').map((eventStream) => {
```

Comment:
Is there a reason why we flatten the EventStreams children but not the Events? Would the results be different if we didn't flatten these here?